### PR TITLE
[FEATURE] GLOB-18938 - moving the logger utils file to nodule-logging

### PR DESCRIPTION
**Pull request by unorthodoxgeek**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,6 +13,7 @@
     "author": "Jason Morita",
```
Comment (cdm-ium) [inline package.json]: This should be `Globality Engineering` now.... The email should be `engineering@globality.com`

**Comment by cdm-ium** [comment]
Can I see a pr against leif or styx to see how this logic looks factored out from those projects?

File: src/utils.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import { anyNonNil } from 'is-uuid';
```
Comment (cdm-ium) [inline src/utils.js]: I'd really prefer a name clearer than `utils`. Maybe `logFormatting`?

Comment (unorthodoxgeek) [reply src/utils.js]: done

**Pull request merged**